static/src/javascripts/projects/common/views/svg.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+export const addClassesAndTitle = (
```

Comment:
sorry to be a pain, but i'd go with `inlineSvg` here i think, instead of `addClassesAndTitle`. it will match https://github.com/guardian/frontend/blob/master/common/app/views/fragments/inlineSvg.scala.html, which this module is designed to ape